### PR TITLE
fix: #1442 signing long message

### PR DESCRIPTION
**Pull request by prybalko**
## Purpose

Fix signing long raw messages.

### CODE REVIEW

File: rust/transaction_signing/src/sign_transaction.rs [hunk 1]
```diff
@@ -147,3 +147,53 @@ pub fn create_signature_png(
...
+
+    use sp_runtime::traits::Verify;
+    use tempfile::tempdir;
+    use transaction_parsing::produce_output;
+
+    #[test]
+    fn sign_long_msg() {
+        let tmp_dir = tempdir().unwrap();
+        populate_cold(tmp_dir.path(), Verifier { v: None }).unwrap();
+        let message = b"<Bytes>baaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaabbb</Bytes>";
```
Comment (montekki) [inline rust/transaction_signing/src/sign_transaction.rs:168]:
```rust
format!("<Bytes>{}bbb</Bytes>", iter::repeat('a').take(256).collect::<String>());
```

Comment (prybalko) [reply rust/transaction_signing/src/sign_transaction.rs]: Nice 👍

**Review by montekki — APPROVED** [review]

**Comment by montekki** [comment]
@prybalko If you've tested with the extension "Sign+Verify" we can merge this

**Comment by prybalko** [comment]
Tested with "Sign+Verify" + PolkadotJs

**Pull request merged**